book/website/collaboration/inclusive-events/inclusive-events-schedules.md

```diff
@@ -0,0 +1,206 @@
+# Inclusive Event Schedules and Agenda
+When developing a schedule or agenda for your event, it is important to reflect on how diferent aspects of an event's schedule will impact how inclusive and accessible your event will be.
+
+As organizers, you should think critically about the timings and design of your event and and how to accommodate the diverse needs of your attendees. This can involve considerations ranging from planning sufficient breaks between sessions, to developing a plan for how to support the engagement and involvement of people with lived experience.
+
+This section provides guidance and reccomendations for how to plan an inclusive and and accessible event schedule (or agenda) and highlights what steps you can take to create a welcoming atmosphere where attendees with lived experience feel heard and valued, and are able to equitable engage in your event.
+
+It is crucial in your planning to prioritize inclusivity and accessibility in your event's scheduling and programming, and to recognize that the many of the traditional approaches to schedules and agendas may create barriers or be inaccessible to people with lived experience or disabilities. 
+
+
+## Event Design: Timing and Event Duration
+When developing your event's program or agenda, you should consider the timing of your event's content. This includes identifying when your event will start and end, the duration of your event's content and how you will structure the event content to make it accessible, and finally the duration of the event itself.
```

Comment:
Typo: This includes identifying when your event will start and end, the duration of your event's content and how you will structure the event**'s** content to make it accessible, and finally the duration of the event itself.
Comment: wondering if itemising might make it easier to read? but it's just a personal impression :)